samples/net/sockets/big_http_download/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _sockets-big-http-download:
+
+Socket Big HTTP Download Example
+################################
+
+Overview
+********
+
+The sockets/big_http_download sample application for Zephyr implements
+a simple HTTP GET client using a BSD Sockets compatible API. Unlike
+the `sockets-http-get` sample application, it downloads a file of
+several megabytes in size, and verifies its integrity using hashing. It
+also performs download repeatedly, tracking the total number of bytes
+transferred. Thus, it can serve as a "load testing" application for
+the Zephyr IP stack.
+
+The source code for this sample application can be found at:
+:file:`samples/net/sockets/big_http_download`.
+
+Requirements
+************
+
+- :ref:`networking_with_qemu`
+- or, a board with hardware networking
+- NAT/routing should be set up to allow connections to the Internet
+- DNS server should be available on the host to resolve domain names
+
+Building and Running
+********************
+
+Build the Zephyr version of the application like this:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/sockets/big_http_download
+   :board: <board_to_use>
+   :goals: build
+   :compact:
+
+``board_to_use`` defaults to ``qemu_x86``. In this case, you can run the
+application in QEMU using ``make run``. If you used another BOARD, you
+will need to consult its documentation for application deployment
+instructions. You can read about Zephyr support for specific boards in
+the documentation at :ref:`boards`.
+
+After the sample starts, it issues an HTTP GET request for
+http://archive.ubuntu.com/ubuntu/dists/xenial/main/installer-amd64/current/images/hd-media/vmlinuz
```

Comment:
So we don't have to change the documentation if you change the specific file being downloaded, I'd recommend changing this to:

    After the sample starts, it issues an HTTP GET request for
    a large file from a public site (configured in the sample's 
    source code) with a published hash value,  showing how reliably
    Zephyr streams non-trival amounts of data, while still taking a
    reasonable amount of time to complete.